libnmstate/prettystate.py

```diff
@@ -15,6 +15,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
+import difflib
```

Comment:
To keep an alphabetic order, it should be below the collections.

Reply:
yes, fixed.